### DISCUSSION TITLE
Looking for a Zepp OS Developer for a Simple Healthcare App

### DISCUSSION THREAD

**Discussion by korea-marketingteacher** [zepp-health/zeppos-forum#398] [General]
Hello,
I am searching for a developer who can build a healthcare app using Zepp OS. The app will serve as an MVP for testing purposes and requires only basic functionality:

Integration of smartwatch data (e.g., oxygen saturation, health metrics) with the application.
Simple UI indicating health status:
Red: Critical
Yellow: Warning
Green: Stable
Skills required:

Experience with Zepp OS and smartwatch data integration.
Ability to design and implement a basic, user-friendly interface.
If you're interested, please reach out to me at fpskceo@gmail.com.

**Comment by zdunex25** [comment]
What would be the logic under?
_Red: Critical
Yellow: Warning
Green: Stable_
We can get spo2, hr, sleep, stress, no sensor for hrv, ecg, sleep apnea tho.
